### DISCUSSION TITLE
Problem with installation of win7-1 machine

### DISCUSSION THREAD

**Discussion by meruemu2** [Marshall-Hallenbeck/red_team_attack_lab#12] [Q&A]
Hello, I have a problem to download the virtual machine win7-1. I installed all the packages and I am working on Ubuntu 21.10. Do you have any ideas why it isn't working ?
Thank you

![Rubygems](https://user-images.githubusercontent.com/94982957/143274833-fac9ef01-2bed-4e30-9ab0-20528bf38072.png)

**Comment by Marshall-Hallenbeck** [comment]
This looks like a Ruby issue within Vagrant. Try downgrading your Ruby version to 1? I think 21.10 is the edge version... try 21.04 or 20.04? Honestly I hate Ruby so I'm only going to be able to give you limited supported with Ruby issues.
